script/DeployMainnet.s.sol

```diff
@@ -84,6 +88,24 @@ contract DeployMainnet is DeployBase {
         config.vettedGateBondCurve.push([2, 1.3 ether]);
         config.vettedGateManager = 0xC52fC3081123073078698F1EAc2f1Dc7Bd71880f; // CSM Committee MS
         config.vettedGateTreeCid = "someCid"; // TODO: to be set in the future
+        // TODO: Set proper values bellow
+        config.vettedGateKeyRemovalCharge = 0.01 ether;
+        config.vettedGateELRewardsStealingAdditionalFine = 0.5 ether;
+        config.vettedGateKeysLimit = type(uint248).max;
+        config.vettedGateAvgPerfLeewayData.push([0, 500]);
+        config.vettedGateRewardShareData.push([0, 10000]);
+        config.vettedGateStrikesLifetimeFrames = 6;
+        config.vettedGateStrikesThreshold = 3;
+        config.vettedGateQueuePriority = 0;
+        config.vettedGateQueueMaxDeposits = 10;
```

Comment:
there is no plan to limit it, it's not exactly the same as from early adoption, so make it unlimit